doc/sphinx/source/user_guide/directories_structure.rst

```diff
@@ -0,0 +1,135 @@
+.. user_guide directories_structure
+
+=====================
+Directories Structure
+=====================
+
+This chapter explains how files and directory are structured in Paparazzi.
+
+
+Here is an extract of the Paparazzi tree:
+
+.. code-block:: text
+
+    ├── conf
+    │   ├── airframes
+    │   │   └── airframe.dtd
+    │   ├── autopilot
+    │   │   └── autopilot.dtd
+    │   ├── conf.xml
+    │   ├── control_panel.xml
+    │   ├── flight_plans
+    │   │   ├── basic.xml
+    │   │   └── flight_plan.dtd
+    │   ├── modules
+    │   │   └── baro_bmp3.xml
+    │   ├── radios
+    │   │   └── FrSkyX9D.xml
+    │   ├── settings
+    │   │   ├── fixedwing_basic.xml
+    │   │   └── settings.dtd
+    │   ├── telemetry
+    │   │   ├── default_fixedwing.xml
+    │   │   ├── default_rotorcraft.xml
+    │   │   └── telemetry.dtd
+    │   ├── tools
+    │   │   ├── blacklisted
+    │   │   ├── gcs.xml
+    │   │   └── messages.xml
+    ├── doc
+    │   └── sphinx
+    │       └── source
+    ├── Makefile
+    ├── paparazzi
+    ├── start.py
+    ├── sw
+    │   ├── ext
+    │   └── airborne
```

Comment:
add `ground_segment/tmtc` `ground_segment/cockpit` and `simulator` folders at least ?